spec/omniauth/strategies/auth0_spec.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
why did you need to add this?

Reply:
`rubocop` suggestion, I don't think it has much effect on what's happening here but idiomatic.